posix_compliance/client.py

```diff
@@ -0,0 +1,72 @@
+# Prepare a NFS-client for posix compliance testing
+# - install needed tools and libs
+# - checkout posix compliance from git
+# - build posix compliance test suite
+# - run the NFSv4.0 tests
+#
+
+import os
+import subprocess
+import time
+import sys
+
+#get the environment variables
+server=os.getenv("SERVER")
+export=os.getenv("EXPORT")
+
+#Install required packages for posix compliance test suite
+cmd = "yum -y install git gcc nfs-utils redhat-rpm-config python-devel krb5-devel perl-Test-Harness *libacl*"
+subprocess.call(cmd, shell=True)
+
+#install posix compliance test suite
+cmd = "rm -rf /root/ntfs-3g-pjd-fstest && git clone https://github.com/ffilz/ntfs-3g-pjd-fstest.git"
+subprocess.call(cmd, shell=True)
+
+#Edit conf file to set fs="ganesha"
+conf_file = "/root/ntfs-3g-pjd-fstest/tests/conf"
+cmd = "sed -i s/'fs=.*'/'fs=\"ganesha\"'/g %s" % conf_file
+subprocess.call(cmd, shell=True)
+
+#Build posix compliance test suite
+posix_home = "/root/ntfs-3g-pjd-fstest/"
```

Comment:
You could set this variable further up, and use it all over the place instead of writing the full path at the previous tasks.